### PR TITLE
Add JSON with historical cumulative distributions

### PR DESCRIPTION
**Pull request by manumonti**
Adding a JSON file with historical data about the total cumulative of T rewards generated on each Merkle distribution facilitates the data collection and processing.

This file just collects the data already included on each distribution folder.

### CODE REVIEW

**Comment by derekpierre** [comment]
Looks great! 

I don't see any changes to a script/new script. Is this file generated as part of the merkle distribution script i.e. it will automatically get updated for every distribution? (this was your original idea @manumonti )

**Comment by manumonti** [comment]
> File looks great!
> 
> I don't see any changes to a script/new script. Is this file generated as part of the merkle distribution script i.e. it will automatically get updated for every distribution? (this was your original idea @manumonti )

Right! Actually, this is a WIP (I forgot to mark the PR as WIP). But we can merge if you need this file ASAP.

**Comment by derekpierre** [comment]
Ah. Not trying to rush you. This is not needed ASAP. The actual distribution of rewards is definitely more important.

File: distributions/distributions.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "CumulativeAmountByDistribution": {
+        "2022-07-15": "124734992091552235627767774",
+        "2022-09-01": "183365680150625789032093616",
```
Comment (derekpierre) [inline distributions/distributions.json:4]: What do you think about adding a "total" to the top of the json file? Kind of like the other MerkleDist.json files. For example,

```
{
    "TotalAmount": "340723072250877984478741713",
    "CumulativeAmountByDistribution": {
        "2022-07-15": "124734992091552235627767774",
        "2022-09-01": "183365680150625789032093616",
    ...
```

That way, there's the overall summary total, and then a breakdown of distributions.

Thoughts?

Comment (derekpierre) [reply distributions/distributions.json:4]: Correct. Feel free to rename `TotalAmount` to something more appropriate (eg. `LastCumulativeAmount` or whatever - you have free reign here) if that would make it clearer.

**Comment by derekpierre** [comment]
> @derekpierre I made a change in the distributions.json.
> 
> Is it Ok from your side to store the numbers as String type? Using Number type in JSON causes the numbers to be stored in Scientific Notation when stringifying the JSON using the new script.

I guess the same string format is done for MerkleDist.json files anyway - so that's fine by me. Whatever is simplest for you.

File: distributions/distributions.json (hunk 1 above)
Comment (manumonti) [reply distributions/distributions.json:4]: Yes, LastCumulativeAmout works for me 👍

Comment (manumonti) [reply distributions/distributions.json]: Done! Could you check the last commit?

File: distributions/distributions.json [hunk 2]
```diff
@@ -1,4 +1,5 @@
 {
+    "LastCumulativeAmount": "340723072250877984478741713",
```
Comment (derekpierre) [inline distributions/distributions.json:2]: 👍

File: README.md [hunk 3]
```diff
@@ -16,6 +16,16 @@ invalidation for every participant.
   - `verify_proof.js`: verify Merkle proof of a distribution.
   - `stake_history.js`: fetch the information of a particular staker, including staking history.
   - `claimed_rewards.js`: calculate the Threshold rewards that has been already claimed.
+- `distributions`:Threshold staking rewards' distributions. Here it is contained the Merkle Root of
+  each distribution and the cumulative rewards earned by each stake.
+  - `YYYY-MM-DD/MerkleDist.json`: includes the Merkle distribution itself: every stake that earned
+    rewards and its Merkle proofs. Also includes the Merkle Root. The amount here shown here is the
```
Comment (derekpierre) [inline README.md:22]:
```suggestion
    rewards and its Merkle proofs. Also includes the Merkle Root. The amount shown here is the
```

File: src/scripts/gen_rewards_dist.js [hunk 4]
```diff
@@ -11,11 +11,11 @@ const MerkleDist = require("../merkle_dist/merkle_dist.js")
...
 const bonusWeight = 0.0
-const preWeight = 0.625
-const tbtcv2Weight = 0.375
-const startTime = 1669852800 // Dec 1st 2022 00:00:00 GMT
-const endTime = 1672531200 // Jan 1st 2023 00:00:00 GMT
-const lastDistribution = "2022-12-01"
+const preWeight = 0.5
+const tbtcv2Weight = 0.5
+const startTime = 1672531200 // Jan 1st 2023 00:00:00 GMT
+const endTime = 1675209600 // Feb 1st 2023 00:00:00 GMT
```
[the remark this answers is not in the document]
Comment (manumonti) [reply src/scripts/gen_rewards_dist.js]: > Out of curiosity, is it possible for you to input dates, and then have the code convert it to a timestamp?

Good idea! We can discuss this in #60 .

File: distributions/distributions.json [hunk 5]
```diff
@@ -0,0 +1,12 @@
+{
+    "LastCumulativeAmount": "340723072250877984478741713",
```
Comment (derekpierre) [inline distributions/distributions.json:2]: Sorry one nit-pick, and I know I provided this name, could we change this to "LatestCumulativeAmount"?

Comment (manumonti) [reply distributions/distributions.json]: Absolutely. I like it. Done

**Review by cygnusv — APPROVED** [review]

**Pull request merged**